master/internal/db/setup.go

```diff
@@ -34,11 +34,10 @@ func authTokenKeypair(ctx context.Context) (*model.AuthTokenKeypair, error) {
 
 // addAuthTokenKeypair adds the new auth token keypair.
 func addAuthTokenKeypair(ctx context.Context, tokenKeypair *model.AuthTokenKeypair) error {
-	newTok := &model.AuthTokenKeypair{
-		PublicKey:  tokenKeypair.PublicKey,
-		PrivateKey: tokenKeypair.PrivateKey,
-	}
-	_, err := Bun().NewInsert().Model(newTok).Exec(ctx)
+	query := `
+	INSERT INTO auth_token_keypair (public_key, private_key)
+	VALUES (:public_key, :private_key)`
+	_, err := Bun().NewRaw(query).Exec(ctx, *tokenKeypair)
```

Comment:
what happened here?

Reply:
I'm running into some errors from `InitAuthKey`, so this was just me testing whether or not keeping it as NewRaw would solve it (it did not).